soh/src/code/z_message_PAL.c

```diff
@@ -828,6 +834,9 @@ u16 Message_DrawItemIcon(PlayState* play, u16 itemId, Gfx** p, u16 i) {
     msgCtx->textPosX += 32;
 
     i++;
+    if (itemId == ITEM_INVALID_4) {
+        i++;
+    }
```

Comment:
need this so we know to skip past the character we're using to get the custom icon index